Projects/Addons/GraphQL/Octopus/CurrentAddonContract.graphql

```diff
@@ -0,0 +1,6 @@
+query CurrentAddonContract($contractId: UUID!) {
```

Comment:
not sure that we even need this one at all but will leave it until we get AddonVariant